packages/bierzo-wallet/src/routes/transactions/components/SendTokens/index.tsx

```diff
@@ -3,10 +3,17 @@ import { useTheme } from "@material-ui/styles";
 import { Block, CircleImage, Typography } from "medulas-react-components";
 import * as React from "react";
 
+import { history } from "../../..";
 import { getBorderColor } from "../../../../theme/css";
-import download from "../../assets/download.svg";
+import { PAYMENT_ROUTE } from "../../../paths";
+import sendIcon from "../../assets/toAddressWhite.svg";
 
-export const CSV_PADDING = 20;
+export const csvPadding = 20;
```

Comment:
There is no `csvPadding` any more, now it is `sendTokenPadding` or something like this.